src/Controller/GithubController.php

```diff
@@ -227,10 +231,13 @@ protected function _augmentDescription($description, $reportId)
     {
         $report = TableRegistry::get('Reports');
         $report->id = $reportId;
+        $incident_count = $this->_getTotalIncidentCount($reportId);
 
         return "$description\n\n\nThis report is related to user submitted report "
             . '[#' . $report->id . '](' . $report->getUrl()
-            . ') on the phpmyadmin error reporting server.';
+            . ') on the phpmyadmin error reporting server.'
+            . 'It, along with its related reports, has been reported **'
+            . $incident_count . '** times.';
```

Comment:
I'd really prefer to have here same table as when linking the issue. Otherwise most of the information is missing when creating new issue.